modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -3670,7 +3670,6 @@
     <xsd:group name="defaultEndpointElements">
         <xsd:sequence>
             <xsd:group ref="commonDefaultEndpointElements"/>
-            <xsd:element ref="abstract-reconnection-strategy" minOccurs="0"/>
```

Comment:
I was talking to Pablo Cabrera and he asks me to create a JIRA to notify these changes.